src/skydb.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const FormData = require("form-data");
+const fs = require("fs");
+const p = require("path");
+
+const { Upload } = require("tus-js-client");
+
+const { buildRequestHeaders, SkynetClient } = require("./client");
+const { defaultOptions, getFileMimeType, makeUrl, walkDirectory, uriSkynetPrefix } = require("./utils");
+
+/**
+ * The tus chunk size is (4MiB - encryptionOverhead) * dataPieces, set in skyd.
+ */
+const TUS_CHUNK_SIZE = (1 << 22) * 10;
```

Comment:
Can you please remove this and all the other tus-related constants and options?

Reply:
Do I need this at all?
I only have this code at the moment:

const defaultSkydbOptions = {
   ...defaultOptions("/skynet/skyfile"),
};

Wanted to remove it first, but nothing worked after that. After that I renamed it to "defaultSkydbOptions" and removed everything except "...defaultOptions("/skynet/skyfile"),". 
This made it work again and everything you suggested is gone.